pastis/matrix_generation/matrix_from_efields.py

```diff
@@ -276,14 +296,14 @@ def __init__(self, which_dm, dm_spec, num_rings=1, initial_path='', saveefields=
         seglist = np.arange(nb_seg) + 1
         self.num_rings = num_rings
         super().__init__(which_dm=which_dm, dm_spec=dm_spec, nb_seg=nb_seg, seglist=seglist, initial_path=initial_path,
-                         saveefields=saveefields, saveopds=saveopds)
+                         saveefields=saveefields, saveopds=saveopds, norm_one_photon=norm_one_photon)
 
     def instantiate_simulator(self):
         optics_input = os.path.join(util.find_repo_location(), 'data', 'SCDA')
         sampling = CONFIG_PASTIS.getfloat('HexRingTelescope', 'sampling')
         self.simulator = HexRingAPLC(optics_input, self.num_rings, sampling)
 
-
+#TODO: add norm_one_photon?
```

Comment:
No, this is not needed if the comment further above about the function signature for `calculate_ref_efield()` and `calculate_ref_efield_wfs()` is followed, so this comment can be deleted.